README.md

```diff
@@ -17,7 +43,7 @@ nvim = nvim_connect("/socket/address")
 
 As a shortcut, `nvim = nvim_env()` will use the address in `$NVIM_LISTEN_ADDRESS`. This is useful to connect to the "parent" nvim instance when running the Julia REPL in a nvim terminal window.
 
-All API methods defined in metadata is defined as corresponding julia functions on the `Neovim` module, except that the `vim_`/`buffer_` prefix is dropped (as the reciever type is identified by the first argument anyway), except for `vim_eval` as `eval` is not overloadable. For instance:
+All API methods defined in metadata is defined as corresponding Julia functions on the `Neovim` module, except that the `vim_`/`buffer_` prefix is dropped (as the receiver type is identified by the first argument anyway), except for `vim_eval` as `eval` is not overloadable. For instance:
```

Comment:
Can also correct the singular verb, so it should be, "All API methods defined in metadata *are* defined . . . ."

May also want to specify/elaborate on what "defined in metadata" means here, since that's not very clear from this paragraph; I think it's simply referring to the API functions that Neovim defines?

Reply:
Yeah, it refers to the remote plugins API. There's a comment in the source suggesting that these functions should be inside another module (`Neovim.API`), which sounds like a good idea, since you could bring them to scope like `import Neovim.API as api`.

I'm also not sure about removing the `vim_` prefix. I get that it's kind of redundant with the methods signatures, but I think consistency with the API docs is more important. I'll see if I open an issue about this.

Reply:
> I'm also not sure about removing the vim_ prefix. I get that it's kind of redundant with the methods signatures, but I think consistency with the API docs is more important. I'll see if I open an issue about this.

Yeah; perhaps it would make more sense if the methods were defined on the structs themselves, so that you could do this sort of thing: `nvim.get_buffers`, `buf.set_line`, etc. That's what I've done in dart-nvim-api and seen in other nvim API bindings (e.g. nvim-rs), and it seems to be the cleanest way in languages with classes/structs/etc.